management_api_app/tests_ma/test_service_bus/test_deployment_status_update.py

```diff
@@ -185,3 +195,54 @@ async def test_state_transitions_from_deployed_to_delete_failed(app, sb_client,
     await receive_message_and_update_deployment(app)
 
     repo().update_item_dict.assert_called_once_with(expected_workspace.dict())
+
+
+@patch('service_bus.deployment_status_update.ResourceRepository')
+@patch('logging.error')
+@patch('service_bus.deployment_status_update.ServiceBusClient')
+@patch('fastapi.FastAPI')
+async def test_outputs_are_added_to_resource_item(app, sb_client, logging_mock, repo):
```

Comment:
could extract the common code into a pytest fixture. I think 2 lines are unique in the other test

Reply:
We spoke, IMO it would make it less readable and we can tackle it at a later stage if needed.